.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**Here are some helpful tips you can follow when submitting a pull request:**
+
+1. Fork [the repository](https://github.com/CircleCI-Public/circleci-cli) and create your branch from `master`.
+2. Run `make build` in the repository root.
+3. If you've fixed a bug or added code that should be tested, add tests!
+4. Ensure the test suite passes (`make test`).
+5. The `--debug` flag is often helpful for debugging HTTP client requests and responses.
+6. Format your code with [gofmt](https://golang.org/cmd/gofmt/).
+7. Make sure your code lints (`make lint`). Tip: `make dev` will install `gometalinter`.
+8. Run `make doc` (after installing `godoc`) to view the API documentation.
+9. Be sure to read our [Contribution Guidelines](./CONTRIBUTING.md).
+10. Find out more about [HACKING](./HACKING.md) the CLI.
```

Comment:
2-8 are super useful, but at this point `HACKING` and this file cover many similar subjects, and it's hard to see a distinction of what should go where. Maybe restructuring this so that points 2. and 5-8. end up in `HACKING`, with a more prominent link to it, is a more maintainable approach?